src/tests/test_boptimizer.cpp

```diff
@@ -136,6 +136,19 @@ struct eval1 {
     }
 };
 
+#ifdef USE_LIBCMAES
+template <typename Params, int obs_size = 1>
+struct eval_bounded {
+    BOOST_STATIC_CONSTEXPR int dim_in = 1;
```

Comment:
why not constexpr? BOOST_STATIC_CONSTEXPR is useful only when we want to compile in pre-c++11.